tests/helper/helper_telemetry.go

```diff
@@ -1,39 +1,56 @@
 package helper
 
 import (
+	"context"
 	"encoding/json"
 	"io/ioutil"
 	"os"
 
 	_ "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
+	"github.com/redhat-developer/odo/pkg/config"
+	envcontext "github.com/redhat-developer/odo/pkg/config/context"
 	"github.com/redhat-developer/odo/pkg/preference"
 	"github.com/redhat-developer/odo/pkg/segment"
 )
 
+const (
+	DebugTelemetryFileEnv = "ODO_DEBUG_TELEMETRY_FILE"
+)
+
 func setDebugTelemetryFile(value string) error {
-	return os.Setenv(segment.DebugTelemetryFileEnv, value)
+	return os.Setenv(DebugTelemetryFileEnv, value)
```

Comment:
Why are you not using envcontext.WithConfig to set this env var?

Reply:
These are integration tests (we are not inside odo), we need to check that odo reads the values from the environment variables